Library/Formula/acpica.rb

```diff
@@ -2,12 +2,12 @@ class Acpica < Formula
   desc "OS-independent implementation of the ACPI specification"
   homepage "https://www.acpica.org/"
   head "https://github.com/acpica/acpica.git"
-  url "https://acpica.org/sites/acpica/files/acpica-unix2-20150204.tar.gz"
+  url "https://acpica.org/sites/acpica/files/acpica-unix2-20150619.tar_1.gz"
   sha1 "8c5514b1171afb40dca40289581c8ba3f17583e1"
 
   bottle do
     cellar :any
-    sha1 "dbb1b43a1702b3eb2855dd4df1ffbb81a4e5e3c1" => :yosemite
+    sha1 "283c594c4b89db8e3bef8fca25021c52938987ad" => :yosemite
```

Comment:
Please don't update this. We'll update this line automatically when we pull the latest version in. You may have intended to update the `sha1` beneath the `url` line up top, which does need changing :wink: